models/valve_landmarks/configs/common.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
Hi @ericspod , in MONAI label, deploy and FL, it is required to have some necessary keys within the train.json and inference.json, for example: `network_def`, `bundle_root` (like in: https://github.com/Project-MONAI/MONAILabel/blob/main/sample-apps/monaibundle/lib/trainers/bundle.py#L30). You can also refer to: https://github.com/Project-MONAI/model-zoo/issues/52 for more details.
These requirements are for bundles in model zoo, thus could we go back to put these common components back? Or do you have any other suggestions? Thanks!

Reply:
I think maybe we should allow for different versions of the same bundle item, with some versions support the preferred keywords and the others less restrictive?

Reply:
It wasn't an issue of keywords being absent, the issue was that multiple config files were needed together, ie. this was in the command line: `-config_file "['configs/inference.json', 'configs/common.json']"` The `common.json` file has common definitions, this eliminated duplication but had the side effect of not testing correctly. I'll propose a feature later but we need some way of including one config file in another with a `_include_` tag or something like that.

Reply:
Hi @ericspod ,

Our `%` syntax supports macro replacement, so actually we can use `"include": "%my/another/config.json"`?

Thanks.

Reply:
@ericspod yes, but the path may be same as the CLI. For more details, please take a look at: https://github.com/Project-MONAI/MONAI/blob/dev/monai/bundle/config_parser.py#L327

Thanks.

Reply:
Thanks Nic, this will work but one would have to refer to components by the key for the macro, eg. "$@include.network_def.to(@include.device)". If might be easier to have a `_include_` tag that would copy values into the current namespace and also give feedback if a circular import is encountered.

Reply:
Interesting, so it's something like `macro replacement` not only on values with `%`, but also on the keys.
CC @wyli @yiheng-wang-nv for more discussion.

Thanks.